contrib/syslog-debun

```diff
@@ -178,6 +189,15 @@ debun_finish_debug () {
 	if [ -d /proc/${pcappid} ]; then
 		kill -INT $pcappid
 	fi
+	if [ -n "$tracing" -a -z "$debug_params" ]; then
+		for i in ${tracepids[*]} ; do
+			kill -INT $i
+		done
+		sleep 2
+		for i in ${tracepids[*]} ; do
+			[ -d /proc/${i} ] && kill -9 $i 2>/dev/null
+		done
```

Comment:
It is good.
Question:
Why do not use loop that try kill the process in repeated cycles if it does not die. This technique is implemented in for example some debian stop scripts. After this it is possible that do not have to implement the same function every time if a process is have to killed.